aroquev/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,12 +30,25 @@
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
-public class DataServlet extends HttpServlet {
-
-  private final ArrayList<String> comments = new ArrayList<String>();  
+public class DataServlet extends HttpServlet { 
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    final ArrayList<String> comments = new ArrayList<String>();
+    for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
+      String comment = (String) entity.getProperty("commentBody");
+      // Timestamp is not used at the moment, but will help in next feature
```

Comment:
`id` is also not used. You should remove the unused variables until they're needed.

Reply:
Done.